### ISSUE TITLE
Support: Only tries to load via HTTPS, PR_END_OF_FILE_ERROR

### ISSUE THREAD

**Issue by keli5** [is-a-dev/register#12053]
### Your subdomain
letters.is-a.dev

### The issue
I just got my domain set up. It uses a CNAME record to point to https://donkeypuncher.chestnut-puffin.ts.net/ . I use LetsEncrypt Certbot to do certs for that domain, and it works fine. I cannot set up HTTPS for my is-a.dev which seems to be preventing me from using it period. When I try to set it up, I get this error.
```
C:\Users\Administrator>certbot certonly --standalone
Saving debug log to C:\Certbot\log\letsencrypt.log
Please enter the domain name(s) you would like on your certificate (comma and/or
space separated) (Enter 'c' to cancel): letters.is-a.dev
Requesting a certificate for letters.is-a.dev

Certbot failed to authenticate some domains (authenticator: standalone). The Certificate Authority reported these problems:
  Domain: letters.is-a.dev
  Type:   connection
  Detail: 2607:f740:f::684: Fetching https://letters.is-a.dev/.well-known/acme-challenge/-e22wU0wqwule08AKpHDsJmV4p7z4cO-EuDMir8Cbtc: Error getting validation data

Hint: The Certificate Authority failed to download the challenge files from the temporary standalone webserver started by Certbot on port 80. Ensure that the listed domains point to this machine and that it can accept inbound connections from the internet.

Some challenges have failed.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile C:\Certbot\log\letsencrypt.log or re-run Certbot with -v for more details.
```
Any help is appreciated!

**Comment by wdhdev** [comment]
We generate SSLs for subdomains I'm pretty sure.

**Comment by keli5** [comment]
I can't access letters.is-a.dev on any device or browser. Firefox on Windows gives me "An error occurred during a connection to letters.is-a.dev. PR_END_OF_FILE_ERROR." Safari on iOS gives me "Safari can't open the page because it couldn't establish a secure connection to the server."

**Comment by keli5** [comment]
The bot that merged the PR mentions that, for server owners, "For HTTPS, you will have to configure an SSL certificate to allow the new subdomain to work."

**Comment by VaibhavSys** [comment]
Does the domain point to your server? And are incoming connections to port 80 allowed and is the certbot temporary server able to listen on it?
